client/my-sites/plugins/plugin-navbar-footer-automattic/index.tsx

```diff
@@ -0,0 +1,35 @@
+import './style.scss';
+import { useTranslate } from 'i18n-calypso';
+
+const PluginNavbarFooterAutomattic = () => {
+	const translate = useTranslate();
+	const automatticRoger = translate(
+		'brainchild contraption creation experiment invention joint medley opus production ruckus thingamajig'
+	);
+
```

Comment:
```suggestion
const [ roger, setRoger ] = useState( '' );

useEffect( () => {
	setRoger( automatticRoger.split( /\s+/ )[ Math.floor( Math.random() * ( 10 - 0 + 1 ) + 0 ) ] );
}, [ automatticRoger ] );
```
We'll want to recalculate if `automatticRoger` changes.